sql_generators/glean_usage/events_first_seen.py

```diff
@@ -0,0 +1,145 @@
+"""Generate Events First Seen table."""
+
+import os
+from collections import namedtuple
+from enum import Enum
+from pathlib import Path
+
+from bigquery_etl.config import ConfigLoader
+from sql_generators.glean_usage.common import (
+    GleanTable,
+    get_table_dir,
+    render,
+    write_sql,
+)
+
+TARGET_TABLE = "events_first_seen"
+VERSION = "v1"
+PREFIX = "events_first_seen"
+PATH = Path(os.path.dirname(__file__))
+
+
+class Browsers(Enum):
+    """Enumeration with browser names and equivalent dataset names."""
+
+    firefox_desktop = "Firefox Desktop"
+
+
+class EventsFirstSeenTable(GleanTable):
+    """Represents generated events_first_seen table."""
+
+    def __init__(self) -> None:
+        """Initialize events_first_seen table."""
+        self.per_app_id_enabled = False
+        self.per_app_enabled = False
+        self.across_apps_enabled = True
+        self.prefix = PREFIX
+        self.target_table_id = f"{TARGET_TABLE}_{VERSION}"
+        self.custom_render_kwargs = {}
+        self.base_table_name = "events_stream_v1"
+
+    def generate_across_apps(
+        self, project_id, apps, output_dir=None, use_cloud_function=True, parallelism=8
+    ):
+        """Generate a query across all apps."""
+        if not self.across_apps_enabled:
+            return
+
+        # output_dir = Path(output_dir)
+        browser = Browsers["firefox_desktop"]
```

Comment:
The enum value doesn't seem to be used anywhere.  Can this be removed?  I don't think this needs to be hardcoded anyway because the app name is in the `apps` dict.